_ont/ont-nokia-g-010g-a.md

```diff
@@ -11,7 +11,7 @@ parent: Nokia
 | -------------- | -------- |
 | Vendor/Brand   | Nokia    |
 | Model          | G-010G-A |
-| Chipset        |          |
+| Chipset        | Broadcom BCM6838  |
 | Flash          |          |
```

Comment:
@drizuid Yes, but you should also fix the whole table by adding spaces. Mine was just a suggestion.